src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -166,15 +165,15 @@ public override string ToString()
         /// </summary>
         /// <param name="parameterFilter">Parameter filter</param>
         /// <returns name="view">View</returns>
-        public Element AddFilter(Revit.Filter.ParameterFilterElement parameterFilter)
+        public Revit.Elements.Views.View AddFilter(Revit.Filter.ParameterFilterElement parameterFilter)
```

Comment:
nothing should happen except that you are avoiding a cast `ToDSType()` when you just return the `View` object which this node works with to begin with. Why the extra cast?